apps/dashboard/src/contract-ui/tabs/nfts/components/token-id.tsx

```diff
@@ -240,6 +245,26 @@ export const TokenIdPage: React.FC<TokenIdPageProps> = ({
                     </GridItem>
                   </>
                 )}
+                <GridItem colSpan={4}>
+                  <Heading size="label.md">Token URI</Heading>
+                </GridItem>
+                <GridItem
+                  colSpan={8}
+                  className="flex flex-row items-center gap-2"
+                >
+                  <AddressCopyButton
+                    size="xs"
+                    address={nft.tokenURI}
+                    title="Token URI"
+                    shortenAddress={true}
+                  />
+                  <Link
```

Comment:
Lets apply the ghost button style on this link by wrapping this with Button component with asChild prop

Reply:
Does this "asChild" prop belong to the new shadcn Button and not tw-components Button? If that's the case I'm happy to revisit this one when I work on replacing the buttons